### PR TITLE
more chemistry

### PR DESCRIPTION
**Pull request by FlapstheMoth**
## About / Why
Ports three chemicals from 13 (Saltpetre, Cyanide and Bath salts)

### CODE REVIEW

**Comment by Merrokitsune** [comment]
IMG_5790

File: Resources/Prototypes/Reagents/narcotics.yml [hunk 1]
```diff
@@ -461,3 +461,54 @@
...
+        type: Remove
+    Medicine:
+      effects:
+      - !type:ResetNarcolepsy
+        conditions:
+        - !type:ReagentThreshold
+          min: 10
+      - !type:ChemVomit
+        probability: 0.2
+        - !type:ReagentThreshold
```
Comment (Merrokitsune) [inline Resources/Prototypes/Reagents/narcotics.yml:513]: There’s something wrong in this particular line.

Comment (FlapstheMoth) [reply Resources/Prototypes/Reagents/narcotics.yml:513]: from /Prototypes/Recipes/Reactions/chemicals.yml... i'll fix it

**Comment by Merrokitsune** [comment]
“Do not merge” label added upon request from PR author. Maybe removed upon PR author request.

**Pull request merged**